### PR TITLE
Small improvements 

### PR DESCRIPTION
**Pull request by evgenysegeda**
- fix versions
- fix build warnings
- fix favicon/web app manifest generation

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ module.exports = {
     siteMetadata: {
         title: `OpenUI5`,
         description: ``,
+        faviconUrl: `/favicon/favicon.png`,
```
Comment (pavelkornev) [inline gatsby-config.js:5]: where is the configuration that favicons are going to genegerated in `favicon/` folder?

Comment (evgenysegeda) [reply gatsby-config.js:5]: This is the static file without background. gatsby-plugin-manifest plugin supports only 1 image for generation. For this reason I used image with white background to generate the all sizes. Favicon is not generated with gatsby-plugin-manifest plugin: line 30

Comment (pavelkornev) [reply gatsby-config.js:5]: As discussed, lets keep all generated icons in `favicon` folder for consistency

Comment (evgenysegeda) [reply gatsby-config.js:5]: done

**Pull request merged**